README.md

```diff
@@ -258,13 +258,13 @@ ros2 service call /controller_manager/switch_controller controller_manager_msgs/
 ```
 Terminal 5 to create a publisher (value is in per mille of torque):
 
-If you are running the demo with one motor:
+If you are running the demo with one motor, send a command in per-mille of rated torque. Gearing is not accounted for.
 ```bash
-ros2 topic pub /forward_torque_controller/commands std_msgs/msg/Float64MultiArray data:\ [100]	
+ros2 topic pub /forward_torque_controller/commands std_msgs/msg/Float64MultiArray data:\ [10]	
```

Comment:
10% of rated torque might be dangerous on some motors. Start with a smaller value.